### DISCUSSION TITLE
Glassmorphic backgrounds

### DISCUSSION THREAD

**Discussion by sasafister** [tailwindlabs/tailwindcss#2884] [Ideas]
Hi guys..As there is more and more need for blurry backgrounds, maybe it's good idea to create some classes for blurry backgrounds... This is definitely a new trend, obvious seeing more and more glassmorphic design everywhere, including need for myself...So this article explains a lot. 

[Glassmorphism in user interfaces](https://uxdesign.cc/glassmorphism-in-user-interfaces-1f39bb1308c9)

I would love to hear what you think. It will be cool to have this natively included in some version of TW.

Keep up with good work guys!

Greets from Croatia

**Comment by MartinHughes-BPC** [comment]
I'd love this too - there's an open PR to implement filters and background-filters: https://github.com/tailwindlabs/tailwindcss/pull/1881

I think Adam is taking the time to think about how best to implement these in Tailwind CSS.

Cheers,
Martin.

**Comment by MathGeniusJodie** [comment]
in the meantime https://github.com/benface/tailwindcss-filters

**Comment by Jinsung-L** [comment]
I built a plugin for CSS filters and backdrop filters.
You can apply multiple filters at once using this plugin.
https://github.com/Jinsung-L/tailwindcss-filters

I think applying the backdrop filter with a white background color and a bit of transparency will work for the glassmorphic design.

**Reply by sasafister** [reply]
Nice work! I will check it out.
